docs/docs/account/user-permissions.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: User Permissions
+description: GrowthBook's User Permissions
+sidebar_label: User Permissions
+slug: user-permissions
+---
+
+GrowthBook supports user roles and permissions at both the global and project level. This allows you to control who can access GrowthBook and what they can do.
+
+When inviting a new user to your organization, you can give the user a global role, which defines their permissions across all projects. If you are enrolled in a `Pro` or `Enterprise` account, you
+have the added ability to give a user project-level roles.
+
+:::note
+
+For instance, you can set a user's global role as `Collaborator`, which allows them to view features and experiements, as well as add
+comments and contribute ideas, and then give them an `Experimenter` role for a particular project, which allows them to create and run experiments, but only for the specified project.
+
+:::
+
+Below are the roles currently available in GrowthBook, ordered from least permissive to most. Each additional level inherits all of the permissions from the roles below it.
```

Comment:
>ordered from least permissive to most.

This isn't correct. `engineer` and `analyst` are non-overlapping permissions.

> Each additional level inherits all of the permissions from the roles below it.

This is confusing. Maybe you mean "above" it?

Reply:
I just took this out - it was confusing either way.

Reply:
I also updated the descriptions of each role, showing what role it inherits permission from explicitly.

Taking a pass at adding a table that has more granular view/edit permissino
s

Reply:
The way I've seen this elsewhere, the axes would be swapped.  One row per feature and the columns are the different roles.

Reply:
Just pushed a commit that introduces the table with the axes as @jdorn described.

I'd love feedback on the rows, along with the descriptors for the various permission levels.